.github/actions/update-docs/action.yml

```diff
@@ -27,8 +27,8 @@ runs:
     - name: Install Python and set up Poetry
       uses: bakdata/ci-templates/actions/python-setup-poetry@v1.5.2
       with:
+        python-version: "3.12"
```

Comment:
Maybe it would be better to run the python actions in the lowest supported python (3.11)

Reply:
what's the advantage?